openshift-knative-operator/hack/008-kafka-ch-dispatcher-revision-history-limit.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml b/knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml
+index 77b11ad0..96d61158 100644
+--- a/knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml
++++ b/knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml
+@@ -911,8 +911,6 @@ spec:
+   # this deployment is going to be scaled up by the
+   # controller when the very first KafkaChannel is created
+   replicas: 0
+-  # remove all old replicasets
+-  revisionHistoryLimit: 0
+   selector:
+     matchLabels:
+       messaging.knative.dev/channel: kafka-channel
```

Comment:
@aliok what was this for?

Reply:
@matzew that was to try if removing old replicasets help. It didn't and I don't think we should include this change when merging.

Reply:
Should we ditch this then?

Reply:
yes, I thnk so - good catch 

I also think the location for this is anyways wrong ... for patching the `knative-operator`, the patch belongs in its `hack` ...